src/model/account/AccountNames.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2018 NEM
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { NamespaceId } from '../namespace/NamespaceId';
+import {Address} from './Address';
+
+/**
+ * Account with linked names
+ */
+export class AccountNames {
+
+    /**
+     *
+     */
+    constructor(
+        /**
+         * Address of the account.
+         */
+        public readonly address: Address,
+        /**
+         * Address linked namespace names
+         */
+        public readonly names: NamespaceId[]) {
```

Comment:
may require a map with `name: id` as it is not possible currently to get the `name` out of a namespace id.

Reply:
Fixed